02-oop-inheritance/js/script.js

```diff
@@ -65,7 +66,24 @@ rocky.unsubscribeObserver(person2);
 var DownloadableMovie = function () {};
 
 DownloadableMovie.prototype = Movie.prototype;
-DownloadableMovie.prototype.download = function () {console.log('downloading')};
+DownloadableMovie.prototype.download = function () {
+    console.log('downloading');
+};
 
 var downloadableMovie = new DownloadableMovie();
 downloadableMovie.download();
+
+var SocialMixin = function () {
+    this.share= function (user) {
```

Comment:
You should always avoid defining functions inside a Class. Also the mixins is an empty object with functions, not an object with properties. Try to re work this out :). I you need some advices please ask me.